drivers/sensor/tenstorrent/pvt/Kconfig

```diff
@@ -0,0 +1,14 @@
+#  Copyright (c) 2025 Tenstorrent AI ULC
+#  SPDX-License-Identifier: Apache-2.0
+
+if SENSOR
```

Comment:
`drivers/sensor/Kconfig` already has an `if SENSOR` condition, so this conditional can be removed